viYarn/AppDelegate.swift

```diff
@@ -17,7 +17,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         window = UIWindow(frame: UIScreen.main.bounds)
-        let rootViewController = ViewController(nibName: nil, bundle: nil)
+        let rootViewController = EpisodeTableViewController(nibName: nil, bundle: nil)
         window?.rootViewController = UINavigationController(rootViewController: rootViewController)
```

Comment:
p0: Let's remove nav controller for now. We don't need to handle it inside VCs. Instead make EpisodesController a rootViewController.